lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -101,7 +101,7 @@ sub _getDirSize {
 
 sub disk_file {
     my ($self) = @_;
-    return locate_asset($self->type, $self->name);
+    return $self->{_location} // ($self->{_location} = locate_asset($self->type, $self->name));
```

Comment:
I think it should work to use =//

Reply:
Right, I'll change that in the other PR first.